README.adoc

```diff
@@ -15,11 +15,11 @@ Manage your SDLC in Spinnaker using the GUI (graphical user interface), or confi
 - https://www.spinnaker.io/reference/providers/[Container deployments to a cloud]
 - https://www.spinnaker.io/guides/user/kubernetes-v2/deploy-manifest/[Container deployments to Kubernetes]
 - https://www.spinnaker.io/docs/concepts/#load-balancer[Load balancers]
-- Security groups
+- https://spinnaker.io/docs/setup/other_config/security/[Security groups]
```

Comment:
A link to Spinnaker configuration seems out of place in a list of user-facing links for Spinnaker concepts. Is this the right link?